po/ja.po

```diff
@@ -14168,38 +14190,49 @@ msgid ""
 "lang.org/std/sync/struct.Mutex.html#impl-Sync-for-Mutex%3CT%3E) blanket "
 "implementation."
 msgstr ""
+"[`impl<T: Send> Sync for Mutex<T>`](https://doc.rust-lang.org/std/sync/"
+"struct.Mutex.html#impl-Sync-for-Mutex%3CT%3E) のブランケット実装があることに"
+"注目してください。"
 
 #: src/concurrency/shared_state/mutex.md:31
 msgid ""
 "`Mutex` in Rust looks like a collection with just one element - the "
 "protected data."
 msgstr ""
+"Rustにおける`Mutex`とは、一つの要素 — 保護されたデータ — からなるコレクション"
+"のようなものです。"
```

Comment:
Translation confidence: Low. We normally don't use — (emdash) in Japanese, so we can also paraphrase the line with parenthesis "（）"?